### PR TITLE
Set command name for user-agent in rest client per component.

### PR DESCRIPTION
**Pull request by mfojtik**
By default, Kubernetes will use the `os.Args[0]` as the source for the command name in `User-Agent` string. This change will allow to override this with custom command name and set this per component name in `hypershift` binary.

That means each component will be uniquely identifiable in prometheus.

### CODE REVIEW

**Comment by deads2k** [comment]
hold my nose and 

/lgtm

File: pkg/cmd/openshift-apiserver/cmd.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/spf13/cobra"
+	"k8s.io/client-go/rest"
```
Comment (sttts) [inline pkg/cmd/openshift-apiserver/cmd.go]: order

File: pkg/cmd/openshift-kube-apiserver/cmd.go [hunk 2]
```diff
@@ -10,6 +10,7 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/spf13/cobra"
+	"k8s.io/client-go/rest"
```
Comment (sttts) [inline pkg/cmd/openshift-kube-apiserver/cmd.go]: order

**Comment by mfojtik** [comment]
@sttts imports order fixed (for other imports as well...)

**Comment by mfojtik** [comment]
@deads2k ^ want to test something new?

**Comment by deads2k** [comment]
this was green.  The integration test issue was found.  merging.

**Pull request merged**